### ISSUE TITLE
[DFC] help of client methods show decorator's help

### ISSUE THREAD

**Issue by chaen** [DIRACGrid/DIRAC#3927]
For example:
```
>> help(dfc.getDirectorySize)
Help on method processWithCheckingArguments in module DIRAC.Resources.Catalog.Utilities:

processWithCheckingArguments(*args, **kwargs) method of DIRAC.Resources.Catalog.FileCatalogClient.FileCatalogClient instance
```

**Comment by fstagni** [comment]
Maybe @andresailer want to look into this?

**Comment by andresailer** [comment]
I will have a look

**Comment by andresailer** [comment]
@chaen: Can you please give me a minimal working example? I.e, what is `dfc` an instance of.
Also which DIRAC version, integration or v6r20, and if this is ipython or plain python interpreter.

**Comment by andresailer** [comment]
Ok, I should have seen you are using the FileCatalogClient, which is fixed in #3928 

However, using the Resources.Catalog.FileCatalog
```python
from DIRAC.Core.Base.Script import parseCommandLine
parseCommandLine()
from DIRAC.Resources.Catalog.FileCatalog import FileCatalog
dfc = FileCatalog()
help(dfc.getDirectorySize)
```
gives
```
Help on method r_execute in module DIRAC.Resources.Catalog.FileCatalog:

r_execute(self, *parms, **kws) method of DIRAC.Resources.Catalog.FileCatalog.FileCatalog instance
    Read method executor.
```

**Comment by fstagni** [comment]
Can we consider this fixed?
